package.json

```
@@ -30,7 +30,8 @@
     "update-service-worker": "node scripts/build/cli updateServiceWorker",
     "-- DEV TEST SCRIPTS": "-----------------------------------------------------------------------------------------------",
     "default-wdio-mobile-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && wdio run ./tests/e2e/default-mobile/wdio.conf.js --suite=all",
-    "e2e-integration-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-e2e-integration",
+    "e2e-integration-all-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-integration-all",
+    "e2e-integration-sentinel-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-integration-sentinel",
```

With the `ci` builds I dropped the `e2e` part because it didn't make any sense to me. Maybe do that here too?

```suggestion
    "integration-all-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-integration-all",
    "integration-sentinel-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-integration-sentinel",
```

---

Cool! Works for me!